burrito_link/.clang-format

```diff
@@ -1,16 +1,19 @@
 ---
 Language: Cpp  # Validate C++
 # BasedOnStyle:  Google
-AccessModifierOffset: -3  # With a tab indent of 4 spaces this equals a 1 space access modifier indent.
-AlignAfterOpenBracket: AlwaysBreak  # Should be `BlockIndent` when it begins to work.
-AlignConsecutiveMacros: false  # Keep the value of a macro close to the macro name.
-AlignConsecutiveAssignments: false  # Keep the new value close to the variable it is being assigned to.
-AlignConsecutiveDeclarations: false  # Keep new variable names close to their type definitions.
-AlignEscapedNewlines: DontAlign  # Keep `\` for escaped newlines close to the last character in the line.
+AccessModifierOffset: -3  # With a tab indent of 4 spaces this equals a 1 space access modifier indent
+AlignAfterOpenBracket: BlockIndent
+AlignConsecutiveMacros: false
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlines: DontAlign
 AlignOperands: true
-AlignTrailingComments: false  # Keep trailing commens close to the last character in the line.
-AllowShortBlocksOnASingleLine: Never  # Force short blocks to have proper newlines.
-AllowShortCaseLabelsOnASingleLine: false  # Force contents of case statements onto the next line.
+AlignTrailingComments: false
+AllowAllArgumentsOnNextLine: true
+AllowAllConstructorInitializersOnNextLine: true  # V
```

Comment:
What is the comment here for?

Reply:
I don't know. I copied the .clang-foramt config from the xml_converter one

Reply:
Looks like it is a comment that lets me know that the setting in that field still needs to be validated. There are similar comments on other rows in this file.